Can't resolve 'vm' in warning from 'asn1.js' since 2.0.1

I'm building a project with `angular`, using `angular-cli` for building. I'm also using `crypto-browserify`, this warning probably comes from it.

Since 2.0.1 this warning appears (ng serve or ng build):

```
WARNING in ./node_modules/asn1.js/lib/asn1/api.js
Module not found: Error: Can't resolve 'vm' in './node_modules/asn1.js/lib/asn1'
```
I don't know explicitly what ng runs internally, but I guess this warning comes from browserify or maybe webpack... not sure.

Downgrading to 2.0.0 the warning disappears.

@angular/cli: 7.3.7  
@angular/core: 7.2.0  
crypto-browserify: 3.12.0

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "bip32",
-  "version": "2.0.1",
+  "version": "2.0.2",
   "description": "A BIP32 compatible library",
   "keywords": [
     "bip32",

--- a/src/crypto.js
+++ b/src/crypto.js
@@ -2,23 +2,20 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 const createHash = require('create-hash');
 const createHmac = require('create-hmac');
-let ripemd160;
-try {
-    ripemd160 = require('crypto')
-        .getHashes()
-        .includes('rmd160')
-        ? 'rmd160'
-        : 'ripemd160';
-}
-catch (err) {
-    ripemd160 = 'rmd160';
-}
 function hash160(buffer) {
-    return createHash(ripemd160)
-        .update(createHash('sha256')
+    const sha256Hash = createHash('sha256')
         .update(buffer)
-        .digest())
         .digest();
+    try {
+        return createHash('rmd160')
+            .update(sha256Hash)
+            .digest();
+    }
+    catch (err) {
+        return createHash('ripemd160')
+            .update(sha256Hash)
+            .digest();
+    }
 }
 exports.hash160 = hash160;
 function hmacSHA512(key, data) {

--- a/ts-src/crypto.ts
+++ b/ts-src/crypto.ts
@@ -1,25 +1,19 @@
 const createHash = require('create-hash');
 const createHmac = require('create-hmac');
 
-let ripemd160: string;
-try {
-  ripemd160 = require('crypto')
-    .getHashes()
-    .includes('rmd160')
-    ? 'rmd160'
-    : 'ripemd160';
-} catch (err) {
-  ripemd160 = 'rmd160';
-}
-
 export function hash160(buffer: Buffer): Buffer {
-  return createHash(ripemd160)
-    .update(
-      createHash('sha256')
-        .update(buffer)
-        .digest(),
-    )
+  const sha256Hash: Buffer = createHash('sha256')
+    .update(buffer)
     .digest();
+  try {
+    return createHash('rmd160')
+      .update(sha256Hash)
+      .digest();
+  } catch (err) {
+    return createHash('ripemd160')
+      .update(sha256Hash)
+      .digest();
+  }
 }
 
 export function hmacSHA512(key: Buffer, data: Buffer): Buffer {

